persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1487,15 +1493,24 @@ tableName = escapeE . tableDBName
 fieldName :: (PersistEntity record) => EntityField record typ -> Text
 fieldName = escapeF . fieldDBName
 
+fieldName' ::  forall record typ. (PersistEntity record, PersistEntityBackend record ~ SqlBackend) => EntityField record typ -> FieldNameDB
```

Comment:
I copied over this method from `MySQL.hs` and it's only used in my new methods; is it worth keeping in?  Or should I give it a new name?